### PR TITLE
Remove Snowbridge teleports

### PR DESCRIPTION
**Pull request by alistair-singh**
Adddresses PR feedback https://github.com/paritytech/polkadot-sdk/pull/2522#discussion_r1430151553

Re-order xcm so we do not need a custom teleporter.

Polkadot-sdk: https://github.com/Snowfork/polkadot-sdk/pull/77

### CODE REVIEW

File: parachain/primitives/router/src/inbound/mod.rs [hunk 1]
```diff
@@ -179,13 +179,14 @@ where
...
 
 		let xcm: Xcm<()> = vec![
-			DescendOrigin(X1(PalletInstance(inbound_queue_pallet_index))),
 			// Teleport required fees.
 			ReceiveTeleportedAsset(total.into()),
 			// Pay for execution.
 			BuyExecution { fees: xcm_fee, weight_limit: Unlimited },
 			// Fund the snowbridge sovereign with the required deposit for creation.
 			DepositAsset { assets: Definite(deposit.into()), beneficiary: bridge_location },
+			// Switch origin.
```
Comment (vgeddes) [inline parachain/primitives/router/src/inbound/mod.rs:188]:
```suggestion
			// Only our inbound-queue pallet is allowed to invoke `UniversalOrigin`
```

File: polkadot-sdk
Comment (vgeddes) [inline polkadot-sdk:1]: Looks like this is pointing to a stale polkadot-sdk

Comment (alistair-singh) [reply polkadot-sdk:1]: No I have double checked this. `main` is stale.

File: parachain/primitives/router/src/inbound/mod.rs (hunk 1 above)
Comment (alistair-singh) [reply parachain/primitives/router/src/inbound/mod.rs]: Addressed.

**Review by vgeddes — APPROVED** [review]
+1

**Review by claravanstaden — APPROVED** [review]

**Pull request merged**